tests/e2e/pageobjects/ide/theia/ContextMenu.ts

```diff
@@ -26,8 +26,8 @@ export class ContextMenu {
      async invokeContextMenuOnTheElementWithMouse(elementLocator: By) {
           Logger.debug(`ContextMenu.invokeContextMenuOnTheElementWithMouse ${elementLocator}`);
 
-          const webElement: WebElement = await this.driverHelper.waitVisibility(elementLocator, TimeoutConstants.TS_CONTEXT_MENU_TIMEOUT);
-          await this.driverHelper.getAction().click(webElement, Button.RIGHT).perform();
+          // const webElement: WebElement = await this.driverHelper.waitVisibility(elementLocator, TimeoutConstants.TS_CONTEXT_MENU_TIMEOUT);
```

Comment:
What was the reason to comment the code?

Reply:
my bad, forgot to uncomment. thanks for noticing

Reply:
welcome